package.json

```diff
@@ -29,8 +29,8 @@
 		"indent-textarea": "^1.0.4",
 		"insert-text-textarea": "^1.0.3",
 		"intervalometer": "^1.0.4",
-		"linkify-issues": "^1.3.0",
-		"linkify-urls": "^2.2.0",
+		"linkify-issues": "^2.0.0",
+		"linkify-urls": "^3.1.0",
```

Comment:
This can't be updated because Firefox doesn't support lookbehind.

https://github.com/sindresorhus/refined-github/issues/803#issuecomment-474172304

Reply:
We should at least update the comment about using the bundled type defs when it's implemented in Firefox and link to the Firefox issue: https://bugzilla.mozilla.org/show_bug.cgi?id=1225665

Reply:
Perhaps v3's types can be backported to v2 and you and you can publish v2.2.1

Reply:
Just checking if we'll add the types to v.2.2.1. If so, I can do this tonight. If not we can leave them in RGH globals.d.ts with the TODO pointing to this comment as @bfred-it mentions, https://github.com/sindresorhus/refined-github/pull/1962#discussion_r277802529

Reply:
> Just checking if we'll add the types to v.2.2.1.

No, just leave them for now.